LibraBFT/Impl/Consensus/SafetyRules/SafetyRules.agda

```diff
@@ -145,6 +151,65 @@ verifyQcM qc = do
 
 ------------------------------------------------------------------------------
 
+consensusState : SafetyRules → ConsensusState
+consensusState self =
+  ConsensusState∙new (self ^∙ srPersistentStorage ∙ pssSafetyData)
+                     (self ^∙ srPersistentStorage ∙ pssWaypoint)
+                     true -- TODO
```

Comment:
Please comment that this is also `TODO` in the Haskell code, and perhaps something about *what* is `TODO`.

Reply:
Looking at the Rust code, I find that this field is not used (except in some tests where this value really doesn't matter).   So, I commented out the field (with a LBFT-OBM-DIFF comment) in the `ConsensusState` type in both Haskell and Agda.  So this `true -- TODO` is now  gone.